lib/services/privacyService.ts

```
@@ -0,0 +1,513 @@
+            return result
+          }
+        } catch (fallbackKeyError) {
+          console.log('Fallback key format also failed:', fallbackKeyError)
+        }
+
+        // Check if this might be legacy base64 encoded data (from migration)
+        try {
+          const decoded = Buffer.from(encryptedData, 'base64').toString('utf8')
+          // Check if it's valid UTF-8 and not binary data by looking for null bytes or excessive control chars
+          const hasNullBytes = decoded.includes('\0')
+          const controlCharCount = (decoded.match(/[\x00-\x08\x0B\x0C\x0E-\x1F\x7F]/g) || []).length
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)